spec/rights_auth_location_spec.rb

```diff
@@ -181,7 +181,7 @@
 
           it 'file is restricted per second access block only' do
             expect(rights).to be_restricted_by_location('location-or-stanford-protected.doc')
-            expect(rights).not_to be_stanford_only_unrestricted_file('location-or-stanford-protected.doc')
+            expect(rights).to be_stanford_only_unrestricted_file('location-or-stanford-protected.doc')
```

Comment:
The distinction between this test and the next one doesn't make any sense to me. Probably worth discussing.

Reply:
the description string "multiple <access> elements (NOT kosher - documenting current behavior)" makes me wonder ... if you are implementing what was missing??   and no longer just documenting current behavior?   The allowance of multipe <access> elements of the same type?   

If so, please change context description.